docs/arm_debugging.md

```diff
@@ -18,7 +18,7 @@ XPM installation instructions can be found [here](https://www.npmjs.com/package/
 
```

Comment:
Would you mind also fixing the typo in the page heading here? :)

Reply:
Oh, I didn't notice at all ! Fixed.